client/src/game/ui/hypothetical.ts

```
@@ -46,6 +46,8 @@ export const start = () => {
     replay.toggleSharedTurns();
   }
 
+  globals.elements.toggleRevealedButton?.setEnabled(true);
```

should be a ! instead of a ? i think

---

we want to error out if the button doesn't exist for whatever reason?

---

i guess not

---

if we want to error out, why does the type allow null/undefined?

---

because its always supposed to exist but you can't instantiate it in the constructor because typescript is dumb